src/index.html

```diff
@@ -7,6 +7,438 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <img
+        class="header__logo"
+        src="images/logo.png"
+        alt="Moyo logo"
+      >
```

Comment:
Consider wrapping the logo image into the link to the home page. It's the basic structure for the logo block to allow the user to get to the home page from everywhere.